style.css

```diff
@@ -0,0 +1,54 @@
+main {
+  margin: auto;
+  width: 400px;
+  text-align: center;
+}
+
+.inpt {
+  display: block;
+  align-self: center;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.input-field {
+  margin: 7px 0 7px 0;
+  border: black 4px solid;
+  width: 300px;
+  padding: 3px;
+}
+
+.button {
+  align-self: flex-end;
+  margin-right: 50px;
+  background-color: white;
+  width: 65px;
+  border: black 2px solid;
+  box-shadow: 4px 2px 2px black;
+}
```

Comment:
- [ ] The `add` button should be on the right side below the input field. Please update to stick with the design.

### Your design

![Screenshot from 2021-11-25 00-00-22](https://user-images.githubusercontent.com/22434660/143295340-60e561f5-9aa0-4b05-a2d3-9f01a77e1cf9.png)

### Original design

![Screenshot from 2021-11-25 00-00-35](https://user-images.githubusercontent.com/22434660/143295327-94acf670-1689-4692-898f-095366f9dc0d.png)

Reply:
This issue has been addressed well. Great job! :+1: :clap: